calliope/core/model.py

```diff
@@ -292,3 +293,41 @@ def info(self):
             times=len(self._model_data.coords.get('timesteps', [])))
         info_strings.append('Model size:   {}'.format(msize))
         return '\n'.join(info_strings)
+
+    def _check_future_deprecation_warnings(self):
```

Comment:
Any reason for moving this into `model.py`?

Reply:
Yeah I believe in terms of clear modularisation it belongs into the Model module: after all, the Model class is the only instance calling the function _and_ it's handing over its internals as a parameter. That's why I concluded, (a) it should be in the same module as the only calling instance, and (b) it should really be a method and not a function.

Reply:
Happy with this change